### PR TITLE
[2.7] Fix unsilenced deprecation notices

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/HttpKernel/Tests/Logger.php [hunk 1]
```diff
@@ -91,7 +91,7 @@ public function debug($message, array $context = array())
      */
     public function emerg($message, array $context = array())
     {
-        trigger_error('Use emergency() which is PSR-3 compatible', E_USER_DEPRECATED);
+        @trigger_error('Use emergency() which is PSR-3 compatible', E_USER_DEPRECATED);
```
Comment (stof) [inline src/Symfony/Component/HttpKernel/Tests/Logger.php]: This is a testing fixture class. Why are we even triggering deprecations here ?

Comment (nicolas-grekas) [inline src/Symfony/Component/HttpKernel/Tests/Logger.php]: It's there since 2.2.0...

**Comment by nicolas-grekas** [comment]
Tests are failing because the current 2.7 is failing

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**